ext/standard/proc_open.c

```diff
@@ -47,16 +47,19 @@
 # if HAVE_PTY_H
 #  include <pty.h>
 # else
-/* Mac OS X defines openpty() in <util.h> */
+/* Mac OS X defines `openpty` in <util.h> */
 #  include <util.h>
 # endif
 #endif
 
 #include "proc_open.h"
 
-static int le_proc_open;
+static int le_proc_open; /* Resource number for `proc` resources */
 
-/* {{{ _php_array_to_envp */
+/* {{{ _php_array_to_envp
+ * Process the `environment` argument to `proc_open`
+ * Convert into data structures which can be passed to underlying OS APIs
+ *   like `exec` on POSIX or `CreateProcessW` on Win32 */
```

Comment:
Here and elsewhere: There should be no indentation for continuation lines. (If you want to make it more clear what is one sentence, add periods at the end of them :)

Reply:
Updating.